examples/matmul/run.cpp

```diff
@@ -799,7 +803,11 @@ void runTest(int version, size_t M, size_t K, size_t N,
   Tensor input = createTensor(ctx, Shape{M, K}, numtype, inputPtr.get());
   Tensor weights = createTensor(ctx, Shape{N, K}, numtype, weightsPtr.get()); // column-major
 
+#ifdef METAL_PROFILER
+  constexpr size_t nIter = 1;
```

Comment:
Replaying gpu grace is slow.
Reduce the number of repetitions to reduce the load.